code/plotting/figED2.R

```diff
@@ -12,6 +12,9 @@ library(mgcv)
 require(gridExtra)
 require(ggplot2)
```

Comment:
suppressPackageStartupMessages() for all packages and change to `library` - `library` is safer than `require`